### ISSUE TITLE
Items.json should only include valid items for speed.

### ISSUE THREAD

**Issue by Detailed** [Detailed/alchr#16]
Items.json currently has every single item in OSRS.  This includes items that can't be traded in the Grand Exchange, like quest items and bank notes.

There are < 8,000 items that are supported in the Grand Exchange.  That means we are parsing through over 4,000 items that will never be used or even looked at since they aren't "valid" anyways.  

Fixing this would mean saving time creating Items and at saving memory.

Basically, items.json should be stripped of all invalid items.

**Comment by Detailed** [comment]
Closed this in #17

**Comment by Detailed** [comment]
Closed this in #17
